### PR TITLE
chore: #6 setup project

### LINKED ISSUE #6: Setups
- Vue.JS
- Hydra
- Custom app
- Database

### PR DESCRIPTION
**Pull request by llorllale**
closes #6 

Signed-off-by: George Aristy <george.aristy@securekey.com>

### CODE REVIEW

**Comment by fqutishat** [comment]
@llorllale we need to build vue inside image https://github.com/trustbloc/edge-adapter/blob/master/images/adapter-rest/Dockerfile#L20

**Comment by llorllale** [comment]
@fqutishat right - I'll make the service serve the frontend

**Comment by llorllale** [comment]
@fqutishat check now.

Note: the vue app is packaged *outside* the docker image. I'd rather avoid issues like trustbloc/edge-agent#173

**Comment by fqutishat** [comment]
> @fqutishat check now.
> 
> Note: the vue app is packaged _outside_ the docker image. I'd rather avoid issues like [trustbloc/edge-agent#173](https://github.com/trustbloc/edge-agent/issues/173)

edge-sandbox need to consume docker image https://github.com/trustbloc/edge-sandbox/tree/master/test/bdd/fixtures/demo

**Comment by fqutishat** [comment]
> > @fqutishat check now.
> > Note: the vue app is packaged _outside_ the docker image. I'd rather avoid issues like [trustbloc/edge-agent#173](https://github.com/trustbloc/edge-agent/issues/173)
> 
> edge-sandbox need to consume docker image https://github.com/trustbloc/edge-sandbox/tree/master/test/bdd/fixtures/demo

how edge-sandbox will use edge-adpater without vue

**Comment by llorllale** [comment]
@fqutishat check now

File: test/bdd/fixtures/adapter-rest/docker-compose.yml [hunk 1]
```diff
@@ -14,6 +14,9 @@ services:
       - ADAPTER_REST_HOST_URL=0.0.0.0:8070
       - ADAPTER_REST_TLS_CACERTS=/etc/tls/ec-cacert.pem
       - ADAPTER_REST_TLS_SYSTEMCERTPOOL=true
+      - ADAPTER_REST_MYSQL_DATASOURCE=mysql://root:secret@tcp(mysql:3306)/mysql
+      - ADAPTER_REST_OP_URL=http://PUT-SOMETHING-HERE.com
+      - ADAPTER_REST_STATIC_FILES=/usr/local/static/adapter-vue
```
Comment (rolsonquadras) [inline test/bdd/fixtures/adapter-rest/docker-compose.yml:19]: Is this property mandatory ? This is already bundled inside docker. Are we going to expect consumers to build their own VueJS project with all JS/CHAPI calls ? Now that i think about it, is adapter-vue really static ?

Comment (fqutishat) [reply test/bdd/fixtures/adapter-rest/docker-compose.yml:19]: I agree not needed

Comment (fqutishat) [reply test/bdd/fixtures/adapter-rest/docker-compose.yml:19]: it's part of the image now, deployer will not need to set it

Comment (llorllale) [reply test/bdd/fixtures/adapter-rest/docker-compose.yml:19]: so should we hardcode the path to the static files in the Dockerfile or in the binary?

Comment (llorllale) [reply test/bdd/fixtures/adapter-rest/docker-compose.yml:19]: > is adapter-vue really static ?

It better be. `adapter-rest` should be the brains of the body, with the frontend just a thin, dumb client. This eases support of flows that do not interact with the wallet.

Comment (llorllale) [reply test/bdd/fixtures/adapter-rest/docker-compose.yml]: @fqutishat @rolsonquadras hardcoded the parameter in the `Dockerfile`, please check

**Pull request merged**